edge/run.sh

```diff
@@ -42,6 +49,9 @@ bashio::log.info "${HASS_TIMEZONE}"
 export HASS_TIMEZONE_2=$(bashio::info "timezone")
 bashio::log.info "${HASS_TIMEZONE_2}"
 
+bashio::log.info "OS Alpine Linux ${ALPINE_VERSION}"
```

Comment:
is this logging relay needed?

Reply:
I'm sorry about this, but it was the only place where I was able to confirm the version of the operating system and python we used in the dockerfile, because it doesn't show in bashio::info, as evidenced in the log below.

```
Starting add-on container...
Wed Jul 31 22:05:14 -03 2024
total 24
drwxr-xr-x    5 root     root          4096 Jul 31 22:05 .
drwxr-xr-x    1 root     root          4096 Jul 31 22:05 ..
drwxr-xr-x    2 root     root          4096 Jul 31 00:40 __pycache__
drwxr-xr-x    2 root     root          4096 Jul 31 00:43 jsons
-rw-------    1 root     root           593 Jul 31 22:05 options.json
drwxr-xr-x    2 root     root          4096 Jul 31 00:40 templates
[22:05:14] INFO: /data/secrets.ini not exists!
Getting mqtt data...
[22:05:15] INFO: null
[22:05:15] INFO: {"supervisor":"2024.06.2","homeassistant":"2024.7.3","hassos":"12.4","docker":"26.1.4","hostname":"homeassistant","operating_system":"Home Assistant OS 12.4","features":["reboot","shutdown","services","network","hostname","timedate","os_agent","haos","resolved","journal","disk","mount"],"machine":"qemux86-64","arch":"amd64","state":"running","supported_arch":["amd64","i386"],"supported":true,"channel":"stable","logging":"info","timezone":"America/Sao_Paulo"}
[22:05:15] INFO: {"supervisor":"2024.06.2","homeassistant":"2024.7.3","hassos":"12.4","docker":"26.1.4","hostname":"homeassistant","operating_system":"Home Assistant OS 12.4","features":["reboot","shutdown","services","network","hostname","timedate","os_agent","haos","resolved","journal","disk","mount"],"machine":"qemux86-64","arch":"amd64","state":"running","supported_arch":["amd64","i386"],"supported":true,"channel":"stable","logging":"info","timezone":"America/Sao_Paulo"}
[22:05:15] INFO: OS Alpine Linux 3.18.0
[22:05:15] INFO: Python 3.10.11
dmslabs - Home Assistant HoyMiles Solar Data Gateway Add-on
```